content/docs/extension-author-guide/create-extension/build-dockerfile.md

```diff
@@ -0,0 +1,80 @@
++++
+title="Generating a build.Dockerfile"
+weight=404
++++
+
+### Examine `tree` extension
+
+#### detect
+
+`cat $workspace/samples/extensions/tree/bin/detect` - the extension always detects (because its exit code is `0`) and provides a dependency
+  called `tree` by writing to the build plan
+
+#### generate
+
+`cat $workspace/samples/extensions/tree/bin/generate` - the extension generates a `build.Dockerfile` that installs `tree` on the builder
+  image
+
+### Re-create our builder with `hello-extensions` updated to require `tree`
+
+1. Edit `$workspace/samples/buildpacks/hello-extensions/bin/detect` to uncomment the first set of lines that
+  output `[[requires]]` to the build plan
+
+2. Create the builder:
+
+```
+$workspace/pack/out/pack builder create $registry_namespace/extensions-builder \
```

Comment:
Should this simply be pack now instead of this workspace path?

Reply:
Good catch! Updated in f7f1ab5e